static/css/s2.css

```diff
@@ -4843,6 +4843,9 @@ body .ui-autocomplete.dictionary-toc-autocomplete .ui-menu-item a.ui-state-focus
   justify-content: center;
   align-items: center;
 }
+.readerNavMenu .content .contentInner.topic-lading-page-content{
+  width: auto;
```

Comment:
I actually think the width needs to be 843px as per the figma. I confirmed this with Mickey